### DISCUSSION TITLE
Help with OpenPlatform

### DISCUSSION THREAD

**Discussion by PanMenel** [MadDeCoDeR/Classic-RBDOOM-3-BFG#157] [Q&A] [answered]
Is there a way to use OpenPlatform in normal RBDOOM-3-BFG, without BFA or do i have to merge them somehow ? Just wanna know if this is possible.

**Comment by MadDeCoDeR** [comment, marked as answer]
The original RBDOOM-3-BFG doesn't
 support Open Platform. Also the developer of RBDOOM-3-BFG is quite against dynamic library usage. I can update Open Platform to support a more static approach. But RBDOOM-3-BFG will require to include a binary of the no Platform implementation in order to even boot. And no, static linking the Steam implementation is not an option (violates the GNU/GPL license)

**Reply by PanMenel** [reply]
Unlucky then, I really don't see a reason why he wouldn't support such an awesome thing, but I guess I will have to bite the bullet and play it without achievements. Thanks for your response, and thanks for your work.
